components/shared/domBuilder.js

```diff
@@ -0,0 +1,12 @@
+import renderToDom from '../../utils/renderToDom';
+
+const domBuilder = () => {
+  const domString = `
+  <div id="navigation"></div>
+  <div id="main-container">
+    
```

Comment:
I feel like we can go ahead and add <div form container>, <div store>, <div view> so that is set up for us when we all start coding and it doesn't get duplicated

Reply:
Sounds good. I'll add these changes!